src/modules/70-pipeline/components/PipelineInputSetForm/StageInputSetForm.tsx

```diff
@@ -779,6 +758,85 @@ export function StageInputSetFormInternal({
               </Container>
             )}
             {hasContainerSecurityContextFields && <Separator topSeparation={16} bottomSeparation={16} />}
+            {(deploymentStageTemplate.infrastructure as any).spec?.priorityClassName && (
+              <Container className={stepCss.bottomMargin3}>
+                <MultiTypeTextField
+                  label={
+                    <Text
+                      font={{ variation: FontVariation.FORM_LABEL }}
+                      margin={{ bottom: 'xsmall' }}
+                      tooltipProps={{ dataTooltipId: 'priorityClassName' }}
+                    >
+                      {getString('pipeline.buildInfra.priorityClassName')}
+                    </Text>
+                  }
+                  name={`${namePath}infrastructure.spec.priorityClassName`}
+                  multiTextInputProps={{
+                    multiTextInputProps: {
+                      expressions,
+                      allowableTypes: allowableTypes
+                    },
+                    disabled: readonly
+                  }}
+                />
+              </Container>
+            )}
+
+            {(deploymentStageTemplate.infrastructure as any).spec?.nodeSelector &&
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232285)
```suggestion
            {(deploymentStageTemplate.infrastructure as unknown).spec?.nodeSelector &&
```